### DISCUSSION TITLE
NotReadableError: Could not start source

### DISCUSSION THREAD

**Discussion by kuhlaid** [mebjas/html5-qrcode#260] [Q&A]
Has anyone else had issues with `NotReadableError: Could not start source` errors? The error crops up using the latest Android mobile (phone) and Chrome browser.

If I try the example at (https://blog.minhazav.dev/research/html5-qrcode#scan-using-file)[https://blog.minhazav.dev/research/html5-qrcode#scan-using-file] I have no problems on Windows 10 (using the Chrome v92) or while using Android mobile. However when I implement this code on my 'production' server it works fine in the Windows 10 environment, but I get the error using Android every time with my production code.

I have noticed that Andriod mobile seems to block a second tab in Chrome from using the camera if another tab is currently using the camera, HOWEVER this does not produce the error in question. I setup a second test of the code at (https://49rvb.csb.app/h5scan.html)[https://49rvb.csb.app/h5scan.html] and it works in both Windows and Android. If I try to load my production code using Android, it throws the error, and then attempting to access one of the examples mentioned above WILL ALSO begin throwing the same error on my Android device.

This error occurs at the point after the browser is allowed access to the camera, and the 'start scanning' button is pressed (no camera image appears). If the scanner is trying to use my front facing camera and I 'sleep' the phone and then access the screen again, the camera image appears in the barcode reader window. I'm guessing this is an Android issue (as I also tested using Firefox for Android, and received an `AbortError: Starting videoinput failed` error using my production code).

**Comment by ROBERT-MCDOWELL** [comment]
I noticed also (sometimes) some lazy reaction from the time you click on start and the time the camera reacts. for this case I start and stop 1 or 2 times and everthing works fine. Where does the issue come from? hard to say, browser? hardware? drivers?
